types/react-dom/server.d.ts

```diff
@@ -72,22 +76,22 @@ export function renderToString(element: ReactElement): string;
  *
  * @deprecated
  */
-export function renderToNodeStream(element: ReactElement): NodeJS.ReadableStream;
+export function renderToNodeStream(element: ReactElement, options?: ServerOptions): NodeJS.ReadableStream;
 
 /**
  * Similar to `renderToString`, except this doesn't create extra DOM attributes
  * such as `data-reactid`, that React uses internally. This is useful if you want
  * to use React as a simple static page generator, as stripping away the extra
  * attributes can save lots of bytes.
  */
-export function renderToStaticMarkup(element: ReactElement): string;
+export function renderToStaticMarkup(element: ReactElement, options?: ServerOptions): string;
 
 /**
  * Similar to `renderToNodeStream`, except this doesn't create extra DOM attributes
  * such as `data-reactid`, that React uses internally. The HTML output by this stream
  * is exactly equal to what `ReactDOMServer.renderToStaticMarkup()` would return.
  */
-export function renderToStaticNodeStream(element: ReactElement): NodeJS.ReadableStream;
+export function renderToStaticNodeStream(element: ReactElement, options?: ServerOptions): NodeJS.ReadableStream;
```

Comment:
```suggestion
export function renderToStaticNodeStream(element: ReactElement): NodeJS.ReadableStream;
```
These are not hydrateable